test/disemvowling_test.js

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai'
+import disemvowling from '../src/disemvowling'
+
+
+describe.only('disemvowling()', function() {
+  it('should be a function', function() {
+    expect(disemvowling).to.be.a('function')
+  })
+
+  context("When given the sentence i am groot", function() {
+    it("should return only the consinants mgrt", function() {
+      expect(disemvowling('i am groot')).to.eql('mgrt')
+    })
+  })
```

Comment:
I would make this context/it combination more concise and focused on the functionality. E.g remove the context and:
```
it('removes all spaces and lowercase vowels from a string', ...
```
and then make your second test testing for upper case letters.